packages/renderer/src/lib/pod/DeployPodToKube.spec.ts

```diff
@@ -325,3 +342,40 @@ test('Fail to deploy ingress if service is not selected', async () => {
   // Expect kubernetesCreateIngress to not be called since we error out as service wasn't selected
   await waitFor(() => expect(kubernetesCreateIngressMock).not.toHaveBeenCalled());
 });
+
+test('Should display Open pod button after successful deployment', async () => {
+  await waitFor(() => kubernetesGetCurrentContextNameMock.mockResolvedValue('default'));
+  await waitRender({});
+  const createButton = screen.getByRole('button', { name: 'Deploy' });
+  expect(createButton).toBeInTheDocument();
+  expect(createButton).toBeEnabled();
+
+  await waitFor(() =>
+    kubernetesCreatePodMock.mockResolvedValue({
+      metadata: { name: 'hello', namespace: 'default' },
+    }),
+  );
+  await waitFor(() =>
+    kubernetesReadNamespacedPodMock.mockResolvedValue({
+      metadata: { name: 'hello' },
+      status: {
+        phase: 'Running',
+      },
+    }),
+  );
+
+  vi.useFakeTimers();
```

Comment:
Is this required ?

Reply:
yes, without using fake timers, it is hard to test these lines:

https://github.com/containers/podman-desktop/pull/4499/files#diff-e8366cdd7002a420e8272fb4522f06900b3ab945fdfc3ded058a5da48448ad03R346-R348